### PR TITLE
updating configure index.md

### PR DESCRIPTION
**Pull request by joecheuk**

### CODE REVIEW

File: _docs/configure/index.md [hunk 1]
```diff
@@ -1,5 +1,48 @@
 ---
-title: Configure 
+title: Configuring Forseti
```
Comment (mirons-google) [inline _docs/configure/index.md]: nit: Forseti -- we're already at /configure/index.html

File: _docs/configure/index.md [hunk 2]
```diff
@@ -1,5 +1,48 @@
...
+    specifying which notifications to enable.
+
+Configurations are centrally maintained in
+forseti-security/configs/server/forseti_server_conf.yaml file that's organized into
+module-specific sections.
+
+## Configuring settings
+
+To set up your configuration, you'll edit
+[forseti_server_conf.yaml.sample](https://github.com/GoogleCloudPlatform/forseti-security/blob/master/configs/server/forseti_server_conf.yaml.sample)
```
Comment (mirons-google) [inline _docs/configure/index.md]: `htmlproofer` can't find this file https://github.com/GoogleCloudPlatform/forseti-security/blob/master/configs/server/forseti_server_conf.yaml.sample

File: _docs/configure/index.md (hunk 1 above)
Comment (joecheuk) [reply _docs/configure/index.md]: Updated to Forseti.

File: _docs/configure/index.md (hunk 2 above)
Comment (joecheuk) [reply _docs/configure/index.md]: Link updated.

File: _docs/configure/index.md [hunk 3]
```diff
@@ -1,5 +1,48 @@
 ---
-title: Configure 
+title: Forseti
 ---
 
-TODO
\ No newline at end of file
+# {{ page.title }}
+
+This page describes how to configure Forseti server once it's set up. Forseti configurations
```
Comment (Tabrarian) [inline _docs/configure/index.md]: I don't think we refer to "Forseti server" anywhere else. Can we just say "Forseti"?

File: _docs/configure/index.md [hunk 4]
```diff
@@ -1,5 +1,48 @@
...
+are global and module-specific settings such as the following:
+
+-   `global`: configurations that are used by multiple modules such as SendGrid API key,
+    GSuite admin account and email recipients.
```
Comment (Tabrarian) [inline _docs/configure/index.md]: Branding: "G Suite"

Comment (Tabrarian) [inline _docs/configure/index.md]: Serial comma for lists of 3 or more: "account, and email recipients."

File: _docs/configure/index.md [hunk 5]
```diff
@@ -1,5 +1,48 @@
...
+-   `scanner`({% link _docs/configure/scanner/index.md %}): configurations that are used only by Forseti Scanner, such as
+    specifying which scanners to enable.
+-   `notifier`({% link _docs/configure/notifier/index.md %}): configurations that are used only by Forseti Notifier, such as
+    specifying which notifications to enable.
+
+Configurations are centrally maintained in
```
Comment (Tabrarian) [inline _docs/configure/index.md]: Missing article: "... in the [filename] file..."

File: _docs/configure/index.md [hunk 6]
```diff
@@ -1,5 +1,48 @@
...
+To set up your configuration, you'll edit
+[forseti_server_conf.yaml.sample](https://github.com/GoogleCloudPlatform/forseti-security/blob/2.0-dev/configs/server/forseti_conf_server.yaml.sample)
+and save it as `forseti_server_conf.yaml`. For convenience, you can maintain different
+versions of this file to support multiple configurations of Forseti.
+
+-   `forseti_server_conf_prod.yaml`
+-   `forseti_server_conf_staging.yaml`
+
+
+### Move Configuration to GCS
```
Comment (Tabrarian) [inline _docs/configure/index.md]: Branding: "Cloud Storage" (this product name is never abbreviated)

File: _docs/configure/index.md [hunk 7]
```diff
@@ -1,5 +1,48 @@
...
+
+If you are running Forseti on GCP, you should copy your edited forseti_conf.yaml to
```
Comment (Tabrarian) [inline _docs/configure/index.md]: Expand the acronym for the first use: "Google Cloud Platform (GCP)"

File: _docs/configure/index.md [hunk 8]
```diff
@@ -1,5 +1,48 @@
...
+your Forseti GCS bucket. When Forseti runs again (via cronjob), it will execute a
```
Comment (Tabrarian) [inline _docs/configure/index.md]: Branding: "Cloud Storage"

File: _docs/configure/index.md [hunk 9]
```diff
@@ -1,5 +1,48 @@
...
+script that downloads the latest conf and rules files.
+
+Use the following commands to copy your conf and rules files to GCS:
```
Comment (Tabrarian) [inline _docs/configure/index.md]: Branding: "Cloud Storage"

File: _docs/configure/index.md [hunk 10]
```diff
@@ -1,5 +1,48 @@
...
+
+```
+gsutil cp configs/forseti_conf.yaml gs://YOUR_FORSETI_GCS_BUCKET/configs/server/forseti_server_conf.yaml
+```
+
+Next, edit your rules and copy the rules directory to the Forseti GCS bucket:
```
Comment (Tabrarian) [inline _docs/configure/index.md]: Branding: "Cloud Storage"

File: _docs/configure/index.md (hunk 8 above)
Comment (Tabrarian) [inline _docs/configure/index.md]: Is this just next time Forseti automatically runs?

File: _docs/configure/index.md [hunk 11]
```diff
@@ -1,5 +1,48 @@
...
+
+```
+gsutil cp -r rules gs://YOUR_FORSETI_GCS_BUCKET/
+```
```
Comment (Tabrarian) [inline _docs/configure/index.md]: Now what? After I finish this step does it mean that my new configuration and rules are now configured for Forseti? We should include a wrap-up paragraph here.

File: _docs/configure/index.md (hunk 6 above)
Comment (Tabrarian) [inline _docs/configure/index.md]: Title consistency: "Moving configuration..."

File: _docs/configure/index.md [hunk 12]
```diff
@@ -1,5 +1,48 @@
...
-TODO
\ No newline at end of file
+# {{ page.title }}
+
+This page describes how to configure Forseti server once it's set up. Forseti configurations
+are global and module-specific settings such as the following:
+
+-   `global`: configurations that are used by multiple modules such as SendGrid API key,
+    GSuite admin account and email recipients.
+-   `scanner`({% link _docs/configure/scanner/index.md %}): configurations that are used only by Forseti Scanner, such as
```
Comment (Tabrarian) [inline _docs/configure/index.md]: Incomplete link, please add brackets.

File: _docs/configure/index.md [hunk 13]
```diff
@@ -1,5 +1,48 @@
...
+    specifying which scanners to enable.
+-   `notifier`({% link _docs/configure/notifier/index.md %}): configurations that are used only by Forseti Notifier, such as
```
Comment (Tabrarian) [inline _docs/configure/index.md]: Incomplete link, please add brackets.

File: _docs/configure/index.md (hunk 7 above)
Comment (Tabrarian) [inline _docs/configure/index.md]: Readability: "..., copy your edited..." (remove extraneous words)

File: _docs/configure/index.md [hunk 14]
```diff
@@ -1,5 +1,48 @@
...
+
+-   `forseti_server_conf_prod.yaml`
+-   `forseti_server_conf_staging.yaml`
+
+
+### Move Configuration to GCS
+
+If you are running Forseti on GCP, you should copy your edited forseti_conf.yaml to
+your Forseti GCS bucket. When Forseti runs again (via cronjob), it will execute a
+script that downloads the latest conf and rules files.
```
Comment (Tabrarian) [inline _docs/configure/index.md]: Why did the rules file come into play here? So far, we've only talked about the conf file - can you add context about why we're copying rules at this point?

File: _docs/configure/index.md (hunk 3 above)
Comment (joecheuk) [reply _docs/configure/index.md]: Updated.

File: _docs/configure/index.md (hunk 4 above)
Comment (joecheuk) [reply _docs/configure/index.md]: Updated

File: _docs/configure/index.md [hunk 15]
```diff
@@ -1,5 +1,42 @@
 ---
-title: Configure 
+title: Forseti
 ---
 
-TODO
\ No newline at end of file
+# {{ page.title }}
+
+This page describes how to configure Forseti once it's set up. Forseti configurations
```
Comment (Tabrarian) [inline _docs/configure/index.md]: after

File: _docs/configure/index.md [hunk 16]
```diff
@@ -1,5 +1,42 @@
...
+
+-   [`global`]: configurations that are used by multiple modules such as SendGrid API key,
+    G Suite admin account, and email recipients.
+-   [`scanner`]({% link _docs/configure/scanner/index.md %}): configurations that are used only by Forseti Scanner, such as
+    specifying which scanners to enable.
+-   [`notifier`]({% link _docs/configure/notifier/index.md %}): configurations that are used only by Forseti Notifier, such as
+    specifying which notifications to enable.
+
+Configurations are centrally maintained in the
+forseti-security/configs/server/forseti_server_conf.yaml file that's organized into
```
Comment (Tabrarian) [inline _docs/configure/index.md]: Add code formatting? `filename`

File: _docs/configure/index.md [hunk 17]
```diff
@@ -1,5 +1,42 @@
...
+and save it as `forseti_server_conf.yaml`. For convenience, you can maintain different
+versions of this file to support multiple configurations of Forseti.
+
+-   `forseti_server_conf_prod.yaml`
+-   `forseti_server_conf_staging.yaml`
+
+
+### Moving Configuration to Google Cloud Storage
+
+If you are running Forseti on GCP, copy your edited forseti_server_conf.yaml to
```
Comment (Tabrarian) [inline _docs/configure/index.md]: Google Cloud Platform (GCP)

File: _docs/configure/index.md [hunk 18]
```diff
@@ -1,5 +1,42 @@
...
+your Forseti GCS bucket. When Forseti runs again (via cronjob), it will execute a
```
Comment (Tabrarian) [inline _docs/configure/index.md]: Cloud Storage

File: _docs/configure/index.md [hunk 19]
```diff
@@ -1,5 +1,42 @@
...
+script that downloads the latest conf and rules files.
+
+Use the following commands to copy your conf and rules files to Google Cloud Storage:
```
Comment (Tabrarian) [inline _docs/configure/index.md]: Cloud Storage

(No "Google" prefix")

File: _docs/configure/index.md (hunk 15 above)
Comment (joecheuk) [reply _docs/configure/index.md]: Updated.

File: _docs/configure/index.md (hunk 16 above)
Comment (joecheuk) [reply _docs/configure/index.md]: Updated.

File: _docs/configure/index.md (hunk 17 above)
Comment (joecheuk) [reply _docs/configure/index.md]: Updated.

File: _docs/configure/index.md (hunk 18 above)
Comment (joecheuk) [reply _docs/configure/index.md]: Updated.

File: _docs/configure/index.md (hunk 19 above)
Comment (joecheuk) [reply _docs/configure/index.md]: Updated.

**Pull request merged**